src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+function convertToNumber(currency) {
+  return Number(currency.split(',').join('.').slice(1));
+}
+
+thead.addEventListener('click', (e) => {
+  if (e.target.tagName !== 'TH') {
+    return;
+  }
+
+  const title = e.target.textContent;
+  const index = e.target.cellIndex;
+
+  const enlargedRows = [...tbody.rows].map((row) => {
+    const valueCell = row.cells[index].textContent;
+
+    switch (title) {
+      case 'Salary':
+        return [convertToNumber(valueCell), row];
+
+      default:
+        return [valueCell, row];
+    }
+  });
```

Comment:
The variable name `enlargedRows` is not very descriptive and can be confusing. Consider renaming it to something more meaningful, such as `rowsWithSortableValues`.